{"uid":"3fc63861b33b4f","name":"test_send_message","fullName":"tests.test_example.ExampleTestCase#test_send_message","historyId":"3bc4f3765c9b31fcf733254dd25c049d","time":{"start":1756971831479,"stop":1756971833975,"duration":2496},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//localhost%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20localhost%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_send_message>\n\n    def test_send_message(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f525f68cf10>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//localhost%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20localhost%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1756971825839,"stop":1756971825839,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1756971833998,"stop":1756971833998,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"pkrvm7jw40e0xgp"},{"name":"thread","value":"2591-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"147eb2590f87d5b3","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//localhost%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20localhost%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1756971830740,"stop":1756971832821,"duration":2081}},{"uid":"3efc2d0f39baec65","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55a13275d08a <unknown>\n#1 0x55a1321fca70 <unknown>\n#2 0x55a1321f3cf1 <unknown>\n#3 0x55a1321e5099 <unknown>\n#4 0x55a1321e6de0 <unknown>\n#5 0x55a1321e543e <unknown>\n#6 0x55a1321e4daa <unknown>\n#7 0x55a1321e4a85 <unknown>\n#8 0x55a1321e28ee <unknown>\n#9 0x55a1321e30ca <unknown>\n#10 0x55a1321fff99 <unknown>\n#11 0x55a13229ad65 <unknown>\n#12 0x55a1322743e2 <unknown>\n#13 0x55a13229a14f <unknown>\n#14 0x55a1322741b3 <unknown>\n#15 0x55a13224059b <unknown>\n#16 0x55a132241971 <unknown>\n#17 0x55a13272225b <unknown>\n#18 0x55a132725fa9 <unknown>\n#19 0x55a132709339 <unknown>\n#20 0x55a132726b58 <unknown>\n#21 0x55a1326edc1f <unknown>\n#22 0x55a13274a118 <unknown>\n#23 0x55a13274a2f6 <unknown>\n#24 0x55a13275c066 <unknown>\n#25 0x7f624769caa4 <unknown>\n#26 0x7f6247729c3c <unknown>","time":{"start":1756971830145,"stop":1756971830544,"duration":399}},{"uid":"8a0b76f3cdefd714","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//localhost%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20localhost%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1756971829268,"stop":1756971831239,"duration":1971}},{"uid":"8291509ea693e22f","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55d4fb2e008a <unknown>\n#1 0x55d4fad7fa70 <unknown>\n#2 0x55d4fad76cf1 <unknown>\n#3 0x55d4fad68099 <unknown>\n#4 0x55d4fad69de0 <unknown>\n#5 0x55d4fad6843e <unknown>\n#6 0x55d4fad67daa <unknown>\n#7 0x55d4fad67a85 <unknown>\n#8 0x55d4fad658ee <unknown>\n#9 0x55d4fad660ca <unknown>\n#10 0x55d4fad82f99 <unknown>\n#11 0x55d4fae1dd65 <unknown>\n#12 0x55d4fadf73e2 <unknown>\n#13 0x55d4fae1d14f <unknown>\n#14 0x55d4fadf71b3 <unknown>\n#15 0x55d4fadc359b <unknown>\n#16 0x55d4fadc4971 <unknown>\n#17 0x55d4fb2a525b <unknown>\n#18 0x55d4fb2a8fa9 <unknown>\n#19 0x55d4fb28c339 <unknown>\n#20 0x55d4fb2a9b58 <unknown>\n#21 0x55d4fb270c1f <unknown>\n#22 0x55d4fb2cd118 <unknown>\n#23 0x55d4fb2cd2f6 <unknown>\n#24 0x55d4fb2df066 <unknown>\n#25 0x7fe46229caa4 <unknown>\n#26 0x7fe462329c3c <unknown>","time":{"start":1756971826709,"stop":1756971827132,"duration":423}},{"uid":"777095af50655619","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55aef536408a <unknown>\n#1 0x55aef4e03a70 <unknown>\n#2 0x55aef4dfacf1 <unknown>\n#3 0x55aef4dec099 <unknown>\n#4 0x55aef4dedde0 <unknown>\n#5 0x55aef4dec43e <unknown>\n#6 0x55aef4debdaa <unknown>\n#7 0x55aef4deba85 <unknown>\n#8 0x55aef4de98ee <unknown>\n#9 0x55aef4dea0ca <unknown>\n#10 0x55aef4e06f99 <unknown>\n#11 0x55aef4ea1d65 <unknown>\n#12 0x55aef4e7b3e2 <unknown>\n#13 0x55aef4ea114f <unknown>\n#14 0x55aef4e7b1b3 <unknown>\n#15 0x55aef4e4759b <unknown>\n#16 0x55aef4e48971 <unknown>\n#17 0x55aef532925b <unknown>\n#18 0x55aef532cfa9 <unknown>\n#19 0x55aef5310339 <unknown>\n#20 0x55aef532db58 <unknown>\n#21 0x55aef52f4c1f <unknown>\n#22 0x55aef5351118 <unknown>\n#23 0x55aef53512f6 <unknown>\n#24 0x55aef5363066 <unknown>\n#25 0x7f39dd49caa4 <unknown>\n#26 0x7f39dd529c3c <unknown>","time":{"start":1756971818052,"stop":1756971818473,"duration":421}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"3fc63861b33b4f.json","parameterValues":[]}